deploy/osd-managed-resources/README.md

```diff
@@ -0,0 +1,15 @@
+# Managed Resources ConfigMap
+
+The included `get-managed-resources.sh` script collects a list of hive-managed resources from a cluster and generates a configmap containing a yaml file that lists out each resource.
+This script should be run periodically to produce an up-to-date reference of all hive-managed resource in a cluster.
+
+## Requirements
+
+* `jq`
+* `pip install oyaml` 
+
+## Instructions to update the ConfigMap
+
+* Ensure you are logged into an OSD cluster
```

Comment:
Does the cluster one is logged into matter? Can a Hive shard be used?

Reply:
Yeah, it matters to be logged into a non-hive OSD AWS or GCP cluster.